src/components/CardRegistration.js

```diff
@@ -0,0 +1,305 @@
+import React, { Component, createRef } from 'react';
+import Joi from 'joi-browser';
+import axios from 'axios';
+
+class CardRegistration extends Component {
+
+    state = {
+        holder: {
+            name: "",
+            home_phone: "",
+            cell_phone: "",
+            work_phone: "",
+            email: "",
+            ssn: "",
+            address: "",
+            po_box: "",
+            zipcode: "",            
+            monthly_income: ""
+            //any additional information goes here
+        },
+        errors: {}
+    };
+
+    checkbox;
+
+    componentDidMount() {
+        this.checkbox = createRef();
+    }
+
+    schema = {
+        name: Joi.string().required().label("Your name"),
+        email: Joi.string().required().email({ minDomainAtons: 2 }).label("Email"),
+        home_phone: Joi.string().min(7).label("Home phone"),
+        cell_phone: Joi.string().min(7).label("Cell phone"),
+        work_phone: Joi.string().min(7).label("Work phone"),
+        address: Joi.string().required().label("Mailing address"),
+        po_box: Joi.string().label("P.O box"),
+        zipcode: Joi.number().required().label("Zipcode"),
+        ssn: Joi.string().min(10).max(10).required().label("Social Security Number"),        

```

Comment:
If I'm not mistaken, isn't SSN 9 digits?

Reply:
SSN is nine digits, 3 area digits 2 Group Number digits and 4 Serial Number digits according to https://www.ssa.gov/history/ssn/geocard.html#:~:text=The%20nine-digit%20SSN%20is%20composed%20of%20three%20parts%3A,set%20of%20four%20digits%20is%20the%20Serial%20Number.

Reply:
I suggest changing the phone numbers to 10 that way area code is always needed, and then ssn can change to min(9).max(9)

Reply:
Made the changes, so it should now use the correct number of digits for SSN and the phone numbers now ask for the area code (10 digits up from 7 digits).